heroku_connect/management/commands/import_mappings.py

```diff
@@ -56,6 +62,32 @@ def handle(self, *args, **options):
         except requests.HTTPError as e:
             raise CommandError("Failed to upload the mapping") from e
 
+        if wait:
+            self.wait_for_import(connection_id)
+
+    @staticmethod
+    def wait_for_import(connection_id):
+        """
+        Wait until connection state is no longer ``IMPORT_CONFIGURATION``.
+
+        Args:
+            connection_id: Heroku Connect connection to monitor.
+
+        Raises:
+            CommandError: If fetch connection information fails.
+
+        """
+        state = utils.ConnectionStates.IMPORT_CONFIGURATION
+        while state == utils.ConnectionStates.IMPORT_CONFIGURATION:
+            # before you get the first state, the API can be a bit behind
+            time.sleep(1)  # breeze
```

Comment:
`breath`?

Reply:
haha, true. I'm still testing. I'll fix it and let you know when I done